templates/contest/contest.html

```diff
@@ -65,6 +65,115 @@
                 {% endtrans %}
             {% endif %}
         </div>
+        {% if contest.show_short_display %}
+            <div id="details">
+                <ul>
+                    {% with authors=contest.authors.all() %}
+                        {% if authors %}
+                            <li>
+                                {% trans trimmed count=authors|length, link_authors=link_users(authors) %}
+                                    The author of this contest is {{ link_authors }}.
+                                {% pluralize count %}
+                                    The authors of this contest are {{ link_authors }}.
+                                {% endtrans %}
+                            </li>
+                        {% endif %}
+                    {% endwith %}
+                    {% with testers=contest.testers.all() %}
+                        {% if testers %}
+                            <li>
+                                {% trans trimmed count=testers|length, link_testers=link_users(testers) %}
+                                    Special thanks to {{ link_testers }} for testing and feedback on the problems.
+                                {% pluralize count %}
+                                    Special thanks to {{ link_testers }} for testing and feedback on the problems.
+                                {% endtrans %}
+                            </li>
+                        {% endif %}
+                    {% endwith %}
+                    <li>
+                        {% if contest.is_rated %}
```

Comment:
Is there a way to clean this up, say in the view using formatted strings?

Reply:
We could break up different segments of that sentence and join them together, but I'm not sure if that will make translations possible/correct. Alternatively, we could use a nested list and list the rating ceiling/floor restrictions there, but I originally opted to not go that route as it may be confusing to a user to see "rated for all participants" followed by some rating restrictions.

Reply:
i18n is pain...

this gets an "ok fine then" from me, I'll see what others have to say on this

Reply:
We definitely must use discrete sentences or colon-style for i18n. Colon-style will be confusing, so I think this is the best we can do.